js/core/rubric-migrations.js

```diff
@@ -66,6 +80,7 @@ const migrations = [
     version: "1.2.0",
     migrations: [
       createCriteriaGroups,
+      fixUndefinedIconPhrase,
```

Comment:
Since the version number will already be 1.2.0, it doesn't seem like this function will actually run when a rubric is fetched. Is that going to be a problem?

Reply:
Good point - I'll fix that.

Reply:
Actually I'll do it in the summary table display PR so I don't have to rebase and push the current follow on PT.